packages/tools/kadena-cli/src/account/types.ts

```diff
@@ -0,0 +1,29 @@
+import type { ChainId } from '@kadena/types';
+import type { INetworkCreateOptions } from '../networks/utils/networkHelpers.js';
+
+export type Predicate = 'keys-all' | 'keys-2' | 'keys-any';
```

Comment:
yes, it can be `string`. But this type currently used for internal type casting because `createPrincipal` method from client utils library currently supports only this `Predicate` type as part of `pred` key. 

I had a chat with @javadkh2 and created a ticket to fix this type https://app.asana.com/0/1205553603119741/1206522684909673/f